packages/atlas/src/components/_inputs/ComboBox/ComboBox.tsx

```diff
@@ -84,11 +86,11 @@ export const ComboBox = <T extends unknown>(props: ComboBoxProps<T>) => {
   return (
     <ComboBoxWrapper ref={comboBoxWrapperRef}>
       <div {...getComboboxProps()}>
-        <TextField
+        <StyledTextField
           {...textFieldProps}
-          error={error}
+          error={error || !!noItemsFound}
           {...getInputProps({ ref: textFieldRef })}
-          nodeEnd={loading && <SvgActionLoader />}
+          nodeEnd={loading && <StyledSpinner size="small" />}
```

Comment:
We shouldn't use Spinner here - I think this is some legacy component, and we should actually remove this. We have a component called Loader and we should use variant `small` here.